src/Servers/Connections.Abstractions/src/PublicAPI/net7.0/PublicAPI.Unshipped.txt

```diff
@@ -1,6 +1,8 @@
 #nullable enable
 Microsoft.AspNetCore.Connections.Features.IStreamClosedFeature
 Microsoft.AspNetCore.Connections.Features.IStreamClosedFeature.OnClosed(System.Action<object?>! callback, object? state) -> void
+Microsoft.AspNetCore.Connections.IConnectionListenerFactorySelector
+Microsoft.AspNetCore.Connections.IConnectionListenerFactorySelector.CanBind(System.Net.EndPoint! endpoint) -> bool
```

Comment:
@dotnet/aspnet-build It looks like we need to [Update our PublicAPI baselines](https://github.com/dotnet/aspnetcore/blob/1d8ffd358c4f65e4e98e474497d9e2cdcc6bfa32/docs/APIBaselines.md#updating-baselines-after-major-releases) now that we're adding new APIs for .NET 8.

We'll have to be careful not to include [IPNetwork.Parse/TryParse](https://github.com/dotnet/aspnetcore/pull/44573/files#diff-7eee3c351b27fb51888f1630847b276dad33accdb9d02dbd46d5d25ba9c7c999) and [RequestLocalizationOptions.CultureInfoUseUserOverride](https://github.com/dotnet/aspnetcore/pull/44282/files#diff-1cd63c3be4af54b57a4feef2a4d91837e1fdf3447318d65d730bdc755ccbc404) in the .NET 7 baseline. I think those are the only APIs we've added since branching for .NET 8.

Reply:
Once 7.0.0 ships, we'll update the baselines. But not until then. That'll involve changes in release/7.0 and copying the Shipped files into main, then patching things up to reflect the new APIs in Unshipped files.